Update index.html

pjunak opened the pull request:
Fixes problems with calculating with some fractions in the binary system. (Page and Tab counting)
Allows more than 10 tabs on page.
Adds multiple tabs to the Preamble page and adds Body page.
Adds function for multilane placeholders.
Adds few minor code-style evening adjustments.

pjunak committed:
Update index.html

Fixes problems with calculating with some fractions in the binary system. (Page and Tab counting)
Allows more than 10 tabs on page.
Adds multiple tabs to the Preamble page and adds Body page.
Adds function for multilane placeholders.
Adds few minor code-style evening adjustments.

pjunak committed:
Update index.html

xsuchy commented:
BuildArch should not be present in resulting spec if nothing is entered in the form. The same for BuildRequires and Requires and ExcludeArch.
Multilines in Requies should not result to:
```
Requires:           fff
fffdd
```
but to
```
Requires:           fff
Requires:       fffdd
````

Description should go to Preamble.


In Body, should be Setup, Build, Install, Clean

pjunak committed:
Update index v2.index

+ Adds multiple tabs to the wizard.
+ Makes results refresh when reaching the last page by any means.
+ Clicking results now select all for easy copy + paste.
+ Makes multiple line entries appear correctly in the results.
+ Makes optional parts of the SPEC truly optional as they won't appear in the results if not filled with anything.

- Removes the refresh button as it is no longer necessary.

! Fixes issue with page and tab addressing.
! Comments starting with # now appear in the spec file as comments.

xsuchy commented:
Thank you.
